docs/core/deploying/native-aot/index.md

```diff
@@ -17,7 +17,7 @@ The Native AOT deployment model uses an ahead-of-time compiler to compile IL to
 
 # [Windows](#tab/windows)
 
-[Visual Studio 2022](https://visualstudio.microsoft.com/vs/), including the **Desktop development with C++** workload with all default components.
+The [Microsoft C++ Build Tools](https://visualstudio.microsoft.com/visual-cpp-build-tools/), or you can install [[Visual Studio 2022](https://visualstudio.microsoft.com/vs/), including the **Desktop development with C++** workload. Open a **Native Tools Command Prompt** to ensure the required tools are on the PATH.
```

Comment:
I don't see this as necessary in my Windows Sandbox clean install and we're going to run vcvarsall to reset the environment anyway (if this is a crossbuild). Already being in the Native Tools command prompt puts us into an [unsupported configuration](https://github.com/dotnet/runtime/issues/115153#issuecomment-2851965800).

```suggestion
The [Microsoft C++ Build Tools](https://visualstudio.microsoft.com/visual-cpp-build-tools/), or you can install [[Visual Studio 2022](https://visualstudio.microsoft.com/vs/), including the **Desktop development with C++** workload.
```